source/index.html.md

```diff
@@ -857,18 +2159,215 @@ var options = {
 	filters: {
 		$title: {
 			$contains: 'adventure'
+		},
+		$timestampUpdated: {
```

Comment:
Please double check it, but I've added the following:

```
var options = {
	filters: {
		$title: {
			$contains: 'adventure'
		},
		coverType: 'hardcover',
		pages: {
			$greaterThan: 200
		}
	}
}
```

I _think_ these satisfy the examples you mentioned... and I think it helps quite a bit more than the previous example.